R/transform-files.R

```diff
@@ -9,17 +9,22 @@
 #' Invisibly returns a data frame that indicates for each file considered for
 #' styling whether or not it was actually changed.
 #' @keywords internal
-transform_files <- function(files, transformers, include_roxygen_examples) {
+transform_files <- function(files,
+                            transformers,
+                            include_roxygen_examples,
+                            root = ".") {
```

Comment:
Can we make sure the new argument `root` is also documented? You can @inheritParams here from `transform_file` and document it there. You can also add a sentence like "required because working directory set with `withr::with_dir()` not respected by `future.apply::future_sapply()`.